src/scripting/player.hpp

```diff
@@ -193,6 +193,11 @@ class Player final
    */
   float get_velocity_y() const;
 
+  /**
+   * Sets the velocity of the player to a programmable/variable speed.
+   */
```

Comment:
Add documentation for params.

Reply:
I noticed `Tux.set_pos()` doesn't have params either, so to keep things consistent i didn't add params here either.

Reply:
It should have param documentation for scripting reference.

Reply:
ok then, ill add params to the function